### PR TITLE
Validate URLs in CLI importer

### LINKED ISSUE #543: `wm import ia` should validate URL before trying to import
I accidentally just swapped around the date and URL arguments and got some funky failed importing. It would be better if the command validated that the URL it was trying to load looks like at actual URL (e.g. starts with `https?://`) and stopped early with a nice error if not.

### PR DESCRIPTION
**Pull request by julianmclain**
This looked like a good first issue, so I took a shot at it. Let me know what you think.

In terms of validation, I wasn't sure exactly what you need. For example, are you only looking at specific TLDs? Is protocol required? etc... 

I decided to use `urlparse` and just confirm that the url has a valid protocol and network location (which apparently doesn't require a TLD being present as I learned).

### CODE REVIEW

**Comment by julianmclain** [comment]
Thanks for the review! Ok, I like the idea of getting rid of `_all_urls_valid()` too.

File: web_monitoring/tests/test_cli.py [hunk 1]
```diff
@@ -48,6 +49,20 @@ def test_filter_unchanged_versions():
...
+    valid_urls = ['https://example.com', 'http://example.com/interesting/path', 'https://subdomain.example.com']
+    valid_url_missing_domain = ['https://google']
+    invalid_single_url = ['asldkfje']
+    invalid_and_valid_urls = ['https://exmaple.com', 'http:badprotocol.com']
+
+    assert all(_is_valid(url) for url in valid_single_url)
+    assert all(_is_valid(url) for url in valid_urls)
+    assert all(_is_valid(url) for url in valid_url_missing_domain)
+    assert not all(_is_valid(url) for url in invalid_single_url)
+    assert not all(_is_valid(url) for url in invalid_and_valid_urls)
```
Comment (Mr0grog) [inline web_monitoring/tests/test_cli.py]: Now that this function isn’t working on a list, I’m not sure it makes sense to be testing lists of things here. It’s fine as-is, but would be nicer if these were just:

```py
assert _is_valid(some_valid_url)
assert not _is_valid(some_invalid_url)
# etc.
```

Comment (julianmclain) [reply web_monitoring/tests/test_cli.py]: Ah yeah, you're right. That's a quick fix, let me tidy up before we merge.

**Comment by julianmclain** [comment]
Ok - fixed the tests. 

I'd love to be listed in the contributors section! I'd also be curious which issue you think would be good to pick up next. `Imports from Wayback list the final redirect URL twice #563` from a few days ago also looked like a good candidate. I'll shoot you a DM in the Archivers Slack.

**Comment by Mr0grog** [comment]
Ugh, sorry it took me a few days to get back around to this. Looks great — I’ll merge and add you to the contributor list.

#563 would be a great one to fix, and should be pretty straightforward. :)

**Pull request merged**